src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,171 @@
+package duke;
+
+import java.util.Scanner;
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
```

Comment:
A very minor coding standard issue, can be easily solved by shifting it all the way down